cap/capture_panic_test.go

```diff
@@ -34,11 +35,23 @@ func TestCapturePanic(t *testing.T) {
 
 		AssertExactMatch(t, events,
 			[]EventP{
-				// start children from left to right
+				// *** Supervisor start process ***
+
 				WorkerStarted("root/child1"),
 				SupervisorStarted("root"),
+				// ^^^ supervisor started successfully, from here, it is supervising the
+				// only worker node it has
 				WorkerFailedWith("root/child1", "Panicking child (1 out of 1)"),
+				// ^^^ As specified with the PanicOnSignalWorker, this worker fails
+				// only one time (1 out of 1)
 				WorkerStarted("root/child1"),
+				// ^^^ Worker is (re)started, this time, it won't panic (given we ran)
```

Comment:
Got it!!! Thank you! I missed the implied restart policy. The policies are awesome, but hard to keep straight.